### ISSUE TITLE
the loss is always  nan

### ISSUE THREAD

**Issue by huangqi111** [MaybeShewill-CV/bisenetv2-tensorflow#53]
4:36:20 Train loss: nan Train miou: nan Val loss: 15.78061 Val miou: 0.00047...

when  train，the loss is always  nan ，how to solve this problem

**Comment by MaybeShewill-CV** [comment]
@huangqi111 Most caused by label. Check label first:)
